Source_Code/TM_Backend/TeamMentor.Users/Users/Users_Creation.cs

```diff
@@ -403,11 +433,25 @@ public static List<int>     createTmUsers               (this TM_UserData userDa
         public static NewUser       with_Random_Data            (this NewUser newUser)
         {
             foreach(var name in newUser.property_Values_AsStrings().keys())
-	            newUser.property(name,"!!10".add_5_RandomLetters());
+                newUser.property(name,"!!10Ma".add_5_RandomLetters());
             newUser.Email = "testUser".random_Email();
             newUser.UserTags = new List<UserTag>();
             newUser.UserTags.add(new UserTag {Key = "key".add_5_RandomLetters(), Value = "value".add_5_RandomLetters()});
             return newUser;
         }
+        public static bool valid_Email_Address(this NewUser tmUser)

```

Comment:
isn't there a class with a number of NewUser extension methods? 
this method should be in there